### ISSUE TITLE
Auto Cache 

### ISSUE THREAD

**Issue by AbilomP12** [HemantKArya/BloomeeTunes#70]
I have an idea ..
If you could make the song cache system..when we heard it should automatically cache and next time it can be able to listen without no internet connection..and not needs to download also

**Comment by HemantKArya** [comment]
Yeah, it would be good idea to enable the caching for song and I was already thought about that feature but due to some problems with YouTube audio's especially the dynamic streaming links, I put it under hold so yeah, I am definitely looking for a way to introduce it soon.
Till then I am open to listen any kind of related suggestions and PRs here in this issue thread.
Thanx for reporting btw.

**Comment by HemantKArya** [comment]
i think the download feature will easily fulfil this need so use the latest version the download feature work stably.
